Source/Core/InputCommon/ControllerInterface/Android/Android.cpp

```diff
@@ -42,18 +42,18 @@ Touchscreen::Touchscreen(int padID) : _padID(padID)
   AddInput(new Button(_padID, ButtonManager::BUTTON_DOWN));
   AddInput(new Button(_padID, ButtonManager::BUTTON_LEFT));
   AddInput(new Button(_padID, ButtonManager::BUTTON_RIGHT));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::STICK_MAIN_LEFT),
-                  new Axis(_padID, ButtonManager::STICK_MAIN_RIGHT));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::STICK_MAIN_UP),
-                  new Axis(_padID, ButtonManager::STICK_MAIN_DOWN));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::STICK_C_LEFT),
-                  new Axis(_padID, ButtonManager::STICK_C_RIGHT));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::STICK_C_UP),
-                  new Axis(_padID, ButtonManager::STICK_C_DOWN));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::TRIGGER_L),
-                  new Axis(_padID, ButtonManager::TRIGGER_L));
-  AddAnalogInputs(new Axis(_padID, ButtonManager::TRIGGER_R),
-                  new Axis(_padID, ButtonManager::TRIGGER_R));
+  AddInput(new Axis(_padID, ButtonManager::STICK_MAIN_LEFT));
+  AddInput(new Axis(_padID, ButtonManager::STICK_MAIN_RIGHT));
+  AddInput(new Axis(_padID, ButtonManager::STICK_MAIN_UP));
+  AddInput(new Axis(_padID, ButtonManager::STICK_MAIN_DOWN));
+  AddInput(new Axis(_padID, ButtonManager::STICK_C_LEFT));
+  AddInput(new Axis(_padID, ButtonManager::STICK_C_RIGHT));
+  AddInput(new Axis(_padID, ButtonManager::STICK_C_UP));
+  AddInput(new Axis(_padID, ButtonManager::STICK_C_DOWN));
+  AddInput(new Axis(_padID, ButtonManager::TRIGGER_L));
+  AddInput(new Axis(_padID, ButtonManager::TRIGGER_L));
+  AddInput(new Axis(_padID, ButtonManager::TRIGGER_R));
+  AddInput(new Axis(_padID, ButtonManager::TRIGGER_R));
```

Comment:
The triggers have been added twice. Same thing on the classic controller.

Reply:
Good catch, was a few of them.